internal/services/controller/controller.go

```diff
@@ -390,28 +392,29 @@ func startConditionalInformers(ctx context.Context,
 
 // collectInitialSnapshot is used to add a time buffer to collect the initial snapshot which is larger than periodic
 // delta because it contains a significant portion of the Kubernetes state.
-func (c *Controller) collectInitialSnapshot(ctx context.Context) error {
-	c.log.Info("collecting initial cluster snapshot")
+func collectInitialSnapshot(ctx context.Context, log logrus.FieldLogger,
+	informers map[string]*custominformers.HandledInformer, queue workqueue.Interface, prepTimeout time.Duration) error {
```

Comment:
Nitpick, move params into separate lines instead of a random new line:
```
func collectInitialSnapshot(
 ctx context.Context, 
 log logrus.FieldLogger,
 informers map[string]*custominformers.HandledInformer, 
 queue workqueue.Interface, 
 prepTimeout time.Duration,
) error {